OpenSim/Simulation/Model/Model.h

```diff
@@ -1004,9 +1001,8 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Model, ModelComponent);
     // Set containing the generalized coordinates in this model.
     CoordinateSet   _coordinateSet;
 
-    // Body used for ground, the inertial frame. This is just a reference
-    // to an existing Body and should not be destructed.
-    Body*           _groundBody;
+    // The ground reference frame
+    Ground _ground;
```

Comment:
Does this mean the FrameSet doesn't own the frames within it? I apologize for not completely following the previous discussions about frames.